### ISSUE TITLE
[Bug Report] Upload component - Trying to validate doc / docx files, but i get empty type from before-upload response. ASAP?

### ISSUE THREAD

**Issue by LG0012** [ElemeFE/element#10813]
### Element UI version
2.3.5

### OS/Browsers version
Chrome Version 65.0.3325.181

### Vue version
2.5.17-beta.0

### Reproduction Link
[https://jsfiddle.net/bu0pccqn/7/](https://jsfiddle.net/bu0pccqn/7/)

### Steps to reproduce
Just select .doc or .docx file, and you will see that type is empty.

Screenshot: https://prnt.sc/j7vvjk

### What is Expected?
application/msword or application/vnd.openxmlformats-officedocument.wordprocessingml.document as a mime type of the selected .doc or .docx file.

### What is actually happening?
I have no idea, thats why im here.. :D

**Comment by Leopoldthecoder** [comment]
![untitled1](https://user-images.githubusercontent.com/10095631/39081464-31c40d68-4574-11e8-9da6-ad6742babffd.gif)

Your demo works for me.

**Comment by LG0012** [comment]
Very strange. Did you tried .docx too?

**Comment by Leopoldthecoder** [comment]
Yeah I did, also works fine. You can try picking a different doc or docx file, or in a different browser or computer to boil this down.

**Comment by LG0012** [comment]
Hi again, I tested on a few different computers, and anyway the file type is empty. So then i tested on the computer which had ms office isntalled, and it worked I got a file type. So I just made a new example, just to make sure that your uploaded is not responsible for this. And still not working for those computers that do not have ms office installed. So I don't know, maybe you have any ideas how to solve that issue? https://jsfiddle.net/64ur0n95/
